### ISSUE TITLE
no scrolling capability on essid page

### ISSUE THREAD

**Issue by bgalletly** [br101/horst#26]
Ive looked and can not figure out how to display all ssids in the terminal interface. In the attached image there was over 3x the amount shown in history and output logs. If this is not a current feature, an option to scroll down so all are visible in live capture would be awesome.

![screenshot - 08172015 - 11 49 10 am](https://cloud.githubusercontent.com/assets/10372978/9310936/bc4bd168-44d8-11e5-8f86-6133f67065bd.jpg)

**Comment by tuomasjjrasanen** [comment]
Yep, confirmed. Would you like to contribute by fixing it? =) Probably other views have the same problem too. I can look at this at some point, but cannot give you any guarantees, I'm really busy at the moment and I have the impression that @br101 is even busier..

**Comment by 8ra1n** [comment]
Ok il give it a go; its about time i started giving back anyways :)

**Comment by bittorf** [comment]
i see your prob, but the solution is easy:
change your font size. a lot if terminals can do this interactively e.g. "CTRL and minus"

**Comment by tuomasjjrasanen** [comment]
I beg to differ with @bittorf. Reducing the size of the terminal font is not a solution, it is just a temporary relief at best. So @8ra1n, I encourage you to solve the issue properly.
